app/app.go

```diff
@@ -464,41 +455,37 @@ func New(
 	// Ethermint keepers
 
 	// Feemarket Keeper
-	feeMarketSs := app.GetSubspace(feemarkettypes.ModuleName)
 	app.FeeMarketKeeper = feemarketkeeper.NewKeeper(
 		appCodec,
 		authtypes.NewModuleAddress(govtypes.ModuleName),
 		keys[feemarkettypes.StoreKey],
 		tkeys[feemarkettypes.TransientKey],
```

Comment:
Subspace is missing on the constructor? Is the guide wrong? https://evm.cosmos.network/docs/v0.4.x/documentation/integration/evm-module-integration#initialize-keepers-critical-order